openmsupply/changes-log
# Getting issues using filters

To test this branch, you can either:

1) (Using the terminal) Run it locally:
- Update the file `.env.example` giving the REPO, OWNER and WEBHOOK_SECRET (which can be your own token from Github user - with access to the repo). Rename the file to `.env`
-  Give parameters as expected. i.e: `node changesLog.js -m 25 -f "Bug: Production, Feature: new, Feature: existing"` where -m 25 is the Milestone number for mSupply version 4.09.00.

2) (Using the terminal) Run it from omsupply-bot: - Require some changes on the branch `36-generate-change-log-for-new-releases` first...
